lib/plane.rb

```diff
@@ -1,2 +1,50 @@
+require_relative 'airport'
+
 class Plane
+
+  DEFAULT_STATUS = "flying"
+
+  attr_accessor :status
+
+  def initialize
+    self.status = DEFAULT_STATUS
+  end
+
+  def land(airport)
+    if airport.land? && self.landed?
+      self.status = "landed"
+    else
+      raise "You are not allowed to land!"
```

Comment:
Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)